modules/tests/src/smithy4s/tests/PizzaSpec.scala

```diff
@@ -141,6 +141,7 @@ abstract class PizzaSpec
         )
       val discriminator = headers.get("X-Error-Type").flatMap(_.headOption)
 
+      expect(headers.get("X-CODE") == Some(List("1"))) &&
```

Comment:
oh wow, that's why I was having trouble writing a test

I was like: damn I can't get the headers out of this client

but I was working in `PizzaClientSpec`, I had not even realize there was a similar one that worked with the http4s client directly, fuuuu